tests/e2e/e2e_test.go

```diff
@@ -69,6 +69,7 @@ var CSITestSuites = []func() framework.TestSuite{
 
 	// Custom test suites specific to Scality S3 CSI driver.
 	customsuites.InitS3MountOptionsTestSuite,
+	customsuites.InitS3MultiVolumeTestSuite,
```

Comment:
There is a typo in the commit message, I will change it post review
It should be `multi volume` and not `multi colume`

Reply:
Pushed with updated commit message